modules/acid/src/main/ts/ephox/acid/api/colour/Transformations.ts

```diff
@@ -0,0 +1,12 @@
+import { Hex, Hsv } from './ColourTypes';
+import * as HexColour from './HexColour';
+import * as HsvColour from './HsvColour';
+import * as RgbaColour from './RgbaColour';
+
+const hex2hsv = (hex: Hex): Hsv => HsvColour.fromRgb(RgbaColour.fromHex(hex));
+const hsv2hex = (hsv: Hsv): Hex => HexColour.fromRgba(RgbaColour.fromHsv(hsv));
+
+export {
```

Comment:
Should this be exported in `Main.ts` as it's in `api` so it should be public normally?